web/controllers/controller_helpers.ex

```diff
@@ -163,4 +163,17 @@ defmodule HexWeb.ControllerHelpers do
     fun = Keyword.get(opts, :fun, fn _, _ -> true end)
     HexWeb.AuthHelpers.authorized(conn, opts, &fun.(conn, &1))
   end
+
+  @doc """
+  Records an entry in audit log. This function should be used within the same transaction
+  as DB operations that are part of the action that is being audited.
+  """
+  def audit(%Plug.Conn{assigns: %{user: user}}, action, opts) do
+    audit(user, action, opts)
+  end
+
+  def audit(user, action, opts) do
```

Comment:
`opts` name here feels confusing to me, it'd be better to name it's `params` as it's what it represents: params for an action (maybe `args`?). :bowtie: